src/Sarif.UnitTests/Writers/SarifLoggerTests.cs

```diff
@@ -169,15 +169,7 @@ public void SarifLogger_WritesSarifLoggerVersion()
                     invocationTokensToRedact: null,
                     invocationPropertiesToLog: null))
                 {
-                    string ruleId = "RuleId";
-                    var rule = new Rule() { Id = ruleId };
-
-                    var result = new Result()
-                    {
-                        RuleId = ruleId
-                    };
-
-                    sarifLogger.Log(rule, result);
+                    LogDefaultResult(sarifLogger);
```

Comment:
>LogDefaultResult [](start = 20, length = 16)

when i say that i aspire to make the world a DRY'er place, I'm not sure I like the way that sounds. You may be the only person in my circle who understands.